### PR TITLE
fix(docs): Add skrub TableReport in the example about the skore project

### LINKED ISSUE #852: fix(docs): Add skrub TableReport in the example about the skore project
Following https://github.com/probabl-ai/skore/issues/806

### PR DESCRIPTION
**Pull request by sylvaincom**
Fix #852

### CODE REVIEW

File: sphinx/conf.py [hunk 1]
```diff
@@ -136,4 +137,4 @@
 
 # Sphinx-Copybutton configuration
 copybutton_prompt_text = r">>> |\.\.\. |\$ "
-copybutton_prompt_is_regexp = True
+copybutton_prompt_is_regexp = True
```
Comment (thomass-dev) [inline sphinx/conf.py:140]: Please revert this change.

Comment (sylvaincom) [reply sphinx/conf.py:140]: I can't seem to solve this

Comment (thomass-dev) [reply sphinx/conf.py]: I fix it.

**Review by thomass-dev — APPROVED** [review]
